build/dependencies.props

```diff
@@ -1,6 +1,13 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
```

Comment:
I'm sure we already considered this but looks like some of these versions are common across repos. Should we keep those ones in some centralized place like KoreBuild or Universe? I think there was a reason but I forget.